libtelemetry/src/main/java/com/mapbox/android/telemetry/MapEventFactory.java

```diff
@@ -21,6 +21,7 @@
   private static final int NO_NETWORK = -1;
   private static final String NOT_A_LOAD_MAP_EVENT_TYPE = "Type must be a load map event.";
   private static final String NOT_A_GESTURE_MAP_EVENT_TYPE = "Type must be a gesture map event.";
+  private static final String NOT_AN_OFFLINE_MAP_EVENT_TYPE = "Type must be an offline map event.";
```

Comment:
Can be removed, since we are no longer worried about type for this event during creation.